modules/virt-configuring-aaq-operator.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.adoc                
+
+:_mod-docs-content-type: PROCEDURE                                    
+[id="virt-configuring-aaq-operator_{context}"]                                   
+= Configuring the AAQ Operator by using the CLI
+
+You can configure the AAQ Operator by specifying the fields of the `spec.applicationAwareConfig` object in the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+* Update the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv --type merge -p '{
+  "spec": {
+    "applicationAwareConfig": {
+      "vmiCalcConfigName": "DedicatedVirtualResources",
+      "namespaceSelector": {
+        "matchLabels": {
+          "app": "my-app"
+        }
+      },
+      "allowApplicationAwareClusterResourceQuota": true
+    }
+  }
+}'
+----
++
+where:
+
+`vmiCalcConfigName`:: Specifies how resource counting is managed for pods that run virtual machine (VM) workloads. Possible values are:
++
+--
+* `VmiPodUsage`: Counts compute resources for pods associated with VMs in the same way as native resource quotas and excludes migration-related resources.
+* `VirtualResources`: Counts compute resources based on the VM specifications, using the VM RAM size for memory and virtual CPUs for processing.
+* `DedicatedVirtualResources` (default): Similar to `VirtualResources`, but separates resource tracking for pods associated with VMs by adding a `/vmi` suffix to CPU and memory resource names. For example, `requests.cpu/vmi` and `requests.memory/vmi`.
+--
+`namespaceSelector`:: Determines the namespaces for which an AAQ scheduling gate is added to pods when they are created. If a namespace selector is not defined, the AAQ Operator targets namespaces with the `application-aware-quota/enable-gating` label as default.
+`allowApplicationAwareClusterResourceQuota`:: If set to `true`, you can create and manage the `ApplicationAwareClusterResourceQuota` object. Setting this attribute to `true` can increase scheduling time.
```

Comment:
```suggestion
`allowApplicationAwareClusterResourceQuota`: If set to `true`, you can create and manage the `ApplicationAwareClusterResourceQuota` object. Setting this attribute to `true` can increase scheduling time.
```

Reply:
The `::` is necessary for asciidoc description list syntax